runtime/src/main/java/org/corfudb/protocols/wireprotocol/PriorityLevel.java

```diff
@@ -0,0 +1,29 @@
+package org.corfudb.protocols.wireprotocol;
+
+import lombok.AllArgsConstructor;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+/**
+ *
+ * Indicates the priority of client requests.
+ *
+ * Created by Maithem on 6/24/19.
+ */
+@AllArgsConstructor
+public enum PriorityLevel {
+    NORMAL(0),
+    HIGH(1);
```

Comment:
Should holeFill messages have HIGH priority?

Reply:
Done.